реалии современного nuget MYP

CyberSturgeon opened the pull request:
+Новая версия пакета
+Адаптация консюмера под CustomerWithAccountMessage
+Адаптация консюмера под CustomerRoleUpdateIdsReportingMessage
+Регистрация консюмера

CyberSturgeon committed:
реалии современного nuget MYP

CyberSturgeon committed:
Завис ноут и не закоммитилась смена версии пакета

Жалко ноут

Negodyaj commented on ReportingService.Application/ServicesConfiguration.cs:
```diff
@@ -35,9 +35,22 @@ public static void ConfigureApplication(this IServiceCollection services, IConfi
+        //            e.Bind(settings.ExchangeName, s =>
+        //            {
+        //                s.RoutingKey = "role.update";
+        //            });
```
а не пора ли разкомментить?

CyberSturgeon replied:
Аппсеттинга с именами очередей и актуальными роутинг ключами нема, Юля будет только к вечеру как я понял, не знаю где взять все названия без неё

CyberSturgeon committed:
HEГP

+AccountService TransactionalAddAsync method
+Обернул две транзакции в транзакцию, привет оперативе, я сильно не хочу делать Unit of Work, need help

CyberSturgeon committed:
wip

CyberSturgeon committed:
pzdc

CyberSturgeon committed:
луч надежды

CyberSturgeon committed:
CustomerMessageConsumer
